test/test_jit.py

```diff
@@ -107,6 +157,27 @@ def get_fn(file_name, script_path):
     return fn
 
 
+def get_execution_plan(graph_executor):
```

Comment:
This is really the debug dumb and not the graph executor itself, right?

Reply:
Yes, I'll change the name to reflect that